server/routes.ts

```diff
@@ -55,10 +55,23 @@ export default function applyRoutes(app: Koa, websocketServer: WebsocketServer)
   // TODO(tec27): we should probably do something based on expected content type as well
   router.get('/robots.txt', send404).get('/favicon.ico', send404)
 
-  // NOTE(tec27): This used to send our feedback URL to clients. Leaving it in place for now to
-  // not break those clients on launch, can be deleted later on.
+  let publicAssetsUrl: string
+  if (process.env.SB_FILE_STORE) {
+    const fileStoreSettings = JSON.parse(process.env.SB_FILE_STORE)
+    if (fileStoreSettings.doSpaces) {
+      const settings = fileStoreSettings.doSpaces
+      publicAssetsUrl = settings.cdnHost
+        ? `https://${settings.cdnHost}/public`
+        : `https://${settings.bucket}.${settings.endpoint}/public`
+    } else {
+      publicAssetsUrl = process.env.SB_CANONICAL_HOST!
```

Comment:
imo you are better off just specifying a relative URL here (`/`)

Reply:
And then I would also make the Spaces ones include a trailing slash for consistency, will make the handling on the client a bit easier

Reply:
Ah sorry, forgot the app wouldn't be on the right origin for that. Doing what you had (including the canonical host) is fine in that case, I think tacking it on in the client is more trouble than it's worth.